src/interactions/dragInteraction.ts

```diff
@@ -70,6 +75,29 @@ export module Interaction {
     }
 
     /**
+     * Returns whether or not this Interaction constrains drag end-points
```

Comment:
When you say end-points, I'm unsure about if this also talks about the point given during the drag sequence rather than the onDragStart/onDragEnd sequences